charts/bandstand-web-service/templates/deployment.yaml

```
@@ -19,7 +19,7 @@ spec:
       serviceAccountName: {{ $relName }}
       containers:
         - name: {{ $relName }}
-          image: {{ .Values.dockerRegistry | default "ktechartifacts-docker.jfrog.io" }}/{{ $relName }}:{{ .Values.image.tag }}
+          image: {{ .Values.dockerRegistry | default "ktechartifacts-docker.jfrog.io" }}/{{ $relName }}:{{ .Values.global.image.tag }}
```

Do we want to allow overriding for a specific chart instance? e.g.
```suggestion
          image: {{ .Values.dockerRegistry | default "ktechartifacts-docker.jfrog.io" }}/{{ $relName }}:{{ .Values.image.tag | default .Values.global.image.tag }}
```

---

That isn't my point. Your change means this can ONLY be read from globals, but can't be specified/overridden for each instance of the chart. Most helm charts I've seen follow this defaulting pattern where values can be set either from the charts values or from the globals.

In this case it probably doesn't matter too much, but just wanted to make you aware of that pattern

---

Yep I'm aware. I can't see a case where we would want to deploy multiple images from the same repo from different git-sha commits. We can still override the global var via circle using `deploy-service` param but images should come from the same git sha. If teams want separate deployments then that should drive them to separate repos.